website/thaliedje/services.py

```
@@ -20,9 +22,64 @@ def user_is_blacklisted(user):
     return ThaliedjeBlacklistedUser.objects.filter(user=user).exists()
 
 
-def has_album_playlist_request_permission(user, player):
-    """Check if a user has album and playlist request permissions."""
-    return user.has_perm("thaliedje.can_request_playlists_and_albums", player)
+def active_thaliedje_control_event(player):
+    """
+    Get the active ThaliedjeControlEvent for a Player.
+
+    :param player: the player
+    :return: the active ThaliedjeControlEvent or None
+    """
+    qs = ThaliedjeControlEvent.objects.filter(player=player.id, active=True)
+    if qs.exists():
+        return qs.first()
+    return None
+
```

There should only be one right? Maybe throw an exception otherwise.